src/_data/contactless/toc.yml

```diff
@@ -0,0 +1,66 @@
+          - text: This PDF provides detailed guidance for transit agency staff on how to complete a User Agreement Scope of Work (SOW) as required to participate in the MSAs for Payment Acceptance Devices (PADs) and Transit Processor Services. Completing an SOW will assist your agency with key scoping decisions as you work toward purchasing hardware and software services.
+          - text: Cal-ITP can assist California transit agencies in drafting and/or reviewing their SOW, making sure it includes your desired operating objectives, intended operating environment, and any desired value-added services. Just send the SOW to hello@calitp.org when it’s ready if you haven’t already connected with Cal-ITP.
+
+      - title: Send the SOW to vendors
+        url: "send-the-sow-to-vendors"
+        paragraphs:
+          - text: When your SOW is ready to send to hardware and software services vendors, you can find their contact information in the MSA User Instructions document (which is linked from each vendor’s contract number). We recommend sending the SOW to all MSA-awarded hardware and software services vendors.
+          - text: You can use <a href="https://docs.google.com/document/d/1CohqgPgu0oePRjFE7HMuLaMpPnEfZla3GdbyeGbAWkg/edit">this email template</a> when sending out your SOW. Be sure to CC hello@calitp.org. Unless otherwise specified by the transit agency, each vendor will respond within five business days of receiving the SOW.
+      - title: Review vendor proposals
+        url: "review-vendor-proposals"
+        paragraphs:
+          - text: Look over the proposals you receive from each hardware and software services vendor. Decide which vendors you want to negotiate with. When you reach an agreement, including any necessary value-added services, you’re ready to place your order for hardware and software services by signing a contract for your purchases.
+      - title: What contract do I sign with the selected vendors?
+        url: "what-contract-do-I-sign-with-selected-vendors"
+        paragraphs:
+          - text: After you’ve finalized the details of your project with the vendor "contractors" you’ve selected, you’ll execute a User Agreement contract and sign it as the “contracting agency.” The User Agreement contract can be the Standard Agreement (<a href="https://www.documents.dgs.ca.gov/dgs/fmc/pdf/std213.pdf">STD 213</a>; note that you'll need Adobe Acrobat or Reader to complete this PDF form) from the California Department of General Services (DGS) or an equivalent contract form, such as this Sample User Agreement (LINK TO WORD DOCUMENT), which can be used by transit providers across the U.S. The User Agreement will incorporate by reference the terms of the MSA and contain the Scope of Work and Payment Provisions.
```

Comment:
Just to confirm, the LINK TO WORD DOCUMENT should be a link to https://docs.google.com/document/d/1sjOre5JFaTJdR1MgP1D0VmbGe4sZmPb5/edit 

Same Q as above, should this be a Google Drive link or a camobilitymarketplace.org/ link?

Reply:
Uploaded: http://resources.calitp.org/mobility-marketplace/User-Agreement-Contract-Form_MSA%205-21-70-28.docx